app/views/organizations/staff/custom_form/questions/_form.html.erb

```diff
@@ -26,7 +29,7 @@
       </div>
 
       <%= f.submit t('general.save'), class: 'btn btn-primary' %>
-      <%= link_to t('general.cancel'), staff_forms_path, class: 'btn btn-secondary' %>
+      <%#= link_to t('general.cancel'), staff_forms_path, class: 'btn btn-secondary' %>
```

Comment:
why the `#`?

Reply:
It was commented while making modular changes for code and missed to uncomment later with correct path. Have corrected it and pushed the code.